core/src/game_state.rs

```diff
@@ -922,8 +924,22 @@ impl PlayPhase {
                     rank: player.level,
                 });
             }
+
+            game_result.insert(
+                player.name.to_string(),
+                PlayerResult {
+                    won_game: landlord_won == self.landlords_team.contains(&player.id),
```

Comment:
nit: should this be `landlord_won == `is_defending`?

Reply:
Ah yeah, I got fixated on the loop to select next landlord but I guess it doesn't have access to `is_defending`